.github/workflows/deploy-apis-to-production.yml

```diff
@@ -1504,4 +1511,25 @@ jobs:
    
       - name: Deploy to K8S
         run: |
-          kubectl apply -f k8s/channel-data/prod-airqo-update-channel-data-job.yaml -n production
\ No newline at end of file
+          kubectl apply -f k8s/channel-data/prod-airqo-update-channel-data-job.yaml -n production
+
+  ### apply nginx ingress ###
+  nginx-ingress:
+    name: apply-nginx-ingress
+    needs: [check]
+    if: needs.check.outputs.run_nginx_ingress == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Login to K8S
+        uses: azure/k8s-set-context@v1
+        with:
+          method: kubeconfig
+          kubeconfig: ${{ secrets.K8S_CONFIG_PROD }}
+
+      - name: Apply ingress changes
+        run: |
+          kubectl apply -f k8s/nginx-ingress/prod-api-ingress-resource.yaml
```

Comment:
[add apply production nginx ingress to workflow](https://github.com/airqo-platform/AirQo-api/pull/1073/commits/49adbb6223196a994b6acaa359aef2e411d4de00)